### ISSUE TITLE
Weird if check in build_forum_jump()

### ISSUE THREAD

**Issue by Sama34** [mybb/mybb#1791]
Now the following piece of code seems confusing to me:
https://github.com/mybb/mybb/blob/feature/inc/functions.php#L2844

Either I don't understand it or the if check is simply plain wrong.

```PHP
function build_forum_jump($pid="0", $selitem="", $addselect="1", $depth="", $showextras="1", $showall=false, $permissions="", $name="fid")
{
	//...
	if($permissions)
	{
		$permissions = $mybb->usergroup;
	}
```

**Comment by Destroy666x** [comment]
I removed it from this and similar functions here: https://github.com/mybb/mybb/pull/1671/files I tried to find out what it does but could not find anything. Porbably outdated code or redundant copy paste leftover.

**Comment by Sama34** [comment]
Yes, seems like some legacy code, removing should cause no issues.

**Comment by PaulBender** [comment]
Fixed in #1671
